woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -243,3 +246,25 @@ def test_url_validate(sample_df):
     actual = logical_type.validate(series, return_invalid_values=True)
     expected = pd.Series({4: "bad_url"}, name="url")
     assert to_pandas(actual).equals(expected)
+
+
+@pytest.mark.parametrize(
+    argnames="logical_type",
+    ids=["age", "age_fractional", "age_nullable"],
+    argvalues=[Age(), AgeFractional(), AgeNullable()],
+)
+def test_age_validate(sample_df, logical_type):
+    invalid_row = pd.Series({4: -3}, name="age", dtype="Int64")
+    series = sample_df["age"]
```

Comment:
I think for testing `AgeFractional` we should pass in a series with a `float64` dtype, since that is what a column idetified as `AgeFractional` will be stored as. Might not matter much, but I think that would be more representative of what would actually happen when validating a dataframe that has that type.

Reply:
sure, it won't affect validation behavior but I don't mind changing